src/guides/v2.4/extension-dev-guide/message-queues/config-mq.md

```diff
@@ -57,9 +57,9 @@ The `name` parameter is required. The topic definition must include either a `re
 Parameter | Description
 --- | ---
 name | A string that uniquely identifies the topic. A topic name should be a series of strings that are separated by periods. The leftmost string should be the most general, and each string afterward should narrow the scope. For example, to describe actions for tending to pets, you might create names such as `cat.white.feed` and `dog.retriever.walk`. Wildcards are not supported in the `communication.xml` file.
-request | Specifies the data type of the topic.
+request | Specifies the data structure that’s used to define a message and is needed when publishing any messages through Magento.
 response | Specifies the format of the response. This parameter is required if you are defining a synchronous topic. Omit this parameter if you are defining an asynchronous topic.
-schema | The interface that describes the structure of the message. The format must be  `<module>\Api\<ServiceName>::<methodName>`.
+schema | Schema describes the message structure. Use the Data Interface format from the Service Contracts. Besides, there is another way to specify a service method signature:`<module>\Api\<ServiceName>::<methodName>` (Magento\Customer\Api\CustomerRepositoryInterface::save).
```

Comment:
```suggestion
schema | Describes the message structure. Use the Data Interface format from the Service Contracts. Besides, there is another way to specify a service method signature:`<module>\Api\<ServiceName>::<methodName>` (Magento\Customer\Api\CustomerRepositoryInterface::save).
```

Reply:
I don't understand "Use the Data Interface format from the Service Contracts. Besides, there is another way to specify a service method signature:"  Is `<module>\Api\<ServiceName>::<methodName>` not the preferred way to specify a schema name? You don't list the data interface format.